forum/qa-include/Q2A/Response.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class Q2A_Response
```

Comment:
mógłby być final

Reply:
smuteczek, że to w core forum.

Reply:
Może smuteczek, ale nie za bardzo widzę opcję, aby pluginem nadpisywać każdą z tych rzeczy tylko po to, aby dołożyć do każdej jedną linijkę z przypisaniem kodu HTTP :P